go.mod

```diff
@@ -8,6 +8,12 @@ require (
 	github.com/taubyte/go-sdk v0.3.4
 )
 
+replace bitbucket.org/taubyte/go-lit => ../go-lit
+
+replace github.com/taubyte/go-sdk => ../go-sdk
```

Comment:
This may cause issues as your current go-sdk branch has a replace for go-sdk-symbols.

Reply:
its fine